dashboard/app/graphs_test.go

```diff
@@ -71,6 +71,11 @@ func TestManagersGraphs(t *testing.T) {
 		Cover:  400,
 	}))
 
+	_, err := c.AuthGET(AccessAdmin, "/test2/graph/fuzzing?Metrics=MaxCorpus&Months=27")
```

Comment:
Need comments explaining the scenario you are testing here.

Reply:
Comment to explain error generation added.

Reply:
testing code was moved into independent functions